.gitignore

```diff
@@ -49,3 +49,7 @@ build/
 
 
 .idea/inspectionProfiles/
+.idea/*
+.idea/markdown-navigator-enh.xml
+.idea/markdown-navigator.xml
+.idea/runConfigurations.xml
```

Comment:
can you please change this to:
```
.idea/*
!.idea/codeStyles
```
We don't wanna to delete the `codeStyle/` folder